docs/source/quickstart.mdx

```diff
@@ -62,6 +68,7 @@ Similarly, you can apply static quantization by simply setting `is_static` to `T
 Static quantization relies on feeding batches of data through the model to estimate the activation quantization parameters ahead of inference time. To support this, 🤗 Optimum allows you to provide a _calibration dataset_. The calibration dataset can be a simple `Dataset` object from the 🤗 Datasets library, or any dataset that's hosted on the Hugging Face Hub. For this example, we'll pick the [`sst2`](https://huggingface.co/datasets/glue/viewer/sst2/test) dataset that the model was originally trained on:
 
 ```python
+>>> from functools import partial
```

Comment:
It does not seem to be used anywhere in the following lines

Reply:
Used when calling the `get_calibration_dataset` method of the quantizer